### PR TITLE
Add support for Rails 6

### PR DESCRIPTION
**Pull request by jacobbednarz**
Updates the gem to support and test against Rails edge.

### CODE REVIEW

**Comment by route** [comment]
@jacobbednarz could you please squash your commits?

File: pjax_rails.gemspec [hunk 1]
```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir['lib/**/*.rb', 'lib/**/*.js', 'vendor/**/*.js']
 
-  s.add_dependency 'railties', '>= 3.2', '< 6.0'
+  s.add_dependency 'railties', '>= 3.2'
```
Comment (route) [inline pjax_rails.gemspec]: I don't think this is necessary, especially when rails 6.0 isn't released

Comment (jacobbednarz) [reply pjax_rails.gemspec]: The gemspec for all the rails components (activerecord, activemodel, etc) are listed as version 6.0.0.alpha which prevents this from being used if left with an upper limit.

**Comment by jacobbednarz** [comment]
Sure, I'll get those tests green and then squash.

**Comment by jacobbednarz** [comment]
@route I've got this pretty much sorted however a failure on rails 3.2 is still outstanding and I can't replicate it locally - https://travis-ci.org/rails/pjax_rails/jobs/459512093#L636. Do you mind having a look and letting me know if you have any ideas? Thanks!

**Comment by jacobbednarz** [comment]
Herrrm, well. That made a liar out of me 🤷‍♂️ Looks like this is now green across the board.

**Comment by route** [comment]
@jacobbednarz thank you, looks good to me now!

**Pull request merged**